### PR TITLE
Mod updates to 1.20.2

### PR DESCRIPTION
**Pull request by watama1299**
FerriteCore, Server Pack Unlocker, Starlight, Blur, Continuity

### CODE REVIEW

File: versions.yml [hunk 1]
```diff
@@ -103,6 +103,20 @@ modGroups:
...
+      modrinthId: uXXizFIs
+      desc: Optimizes memory usage
+      url: https://cdn.modrinth.com/data/uXXizFIs/versions/FCnCG6PS/ferritecore-6.0.0-fabric.jar
+
+    - name: Starlight
+      license: LGPL-3
+      homepage: https://modrinth.com/mod/starlight
+      modrinthId: H8CaAYZC
+      desc: Rewrites light engine to reduces chunk load stuttering
+      url: https://cdn.modrinth.com/data/H8CaAYZC/versions/98VOoYPX/starlight-1.1.2%2Bfabric.bdaeb21.jar
```
Comment (kristofbolyai) [inline versions.yml:118]: This is only reported as 1.20-1.20.1 on Modrinth. Does it work on 1.20.2?

File: versions.yml [hunk 2]
```diff
@@ -188,6 +202,12 @@ modGroups:
       desc: Adds Texture Features to entities in Optifine compatible format.
       url: https://cdn.modrinth.com/data/BVzZfTc1/versions/1Y3lxOX6/entity_texture_features_fabric_1.20.2-4.5.1.jar
 
+    - name: Continuity
```
Comment (kristofbolyai) [inline versions.yml:205]: I know it was missing before too but Indium is a dependency. Can you add it?

File: versions.yml [hunk 3]
```diff
@@ -202,6 +222,13 @@ modGroups:
       url: https://mediafilez.forgecdn.net/files/4745/844/ShoulderSurfing-Fabric-1.20.1-2.5.jar
       requires: [Forge Config API Port]
 
+    - name: Blur
```
Comment (kristofbolyai) [inline versions.yml:225]: Probably a HUD-Menu enchancement?

File: versions.yml [hunk 4]
```diff
@@ -213,6 +240,12 @@ modGroups:
       url: https://cdn.modrinth.com/data/Bh37bMuy/versions/UpKbnidp/reeses_sodium_options-1.6.5%2Bmc1.20.2-build.96.jar
       requires: [Sodium]
 
+    - name: Server Packer Unlocker
```
Comment (kristofbolyai) [inline versions.yml:243]: I would put this as a misc but check where it was before, and keep it there.

File: versions.yml (hunk 1 above)
Comment (watama1299) [reply versions.yml:118]: I have not personally tested it but Modrinth does say it's for 1.20.2. Screenshot below.

image

Comment (kristofbolyai) [reply versions.yml:118]: Ive missed that

File: versions.yml (hunk 3 above)
Comment (watama1299) [reply versions.yml:225]: Seems like Blur was under Misc previously, but I can put it under HUD enchancement if that's better.

**Pull request closed without merging**